openedx/features/job_board/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.conf.urls import url
 
-from .views import JobDetailView, JobListView
+from .views import JobCreateView, JobDetailView, JobListView
 
 urlpatterns = [
-    url(r'^jobs/$', JobListView.as_view(), name='job_list'),
-    url(r'^jobs/(?P<pk>[0-9]+)/$', JobDetailView.as_view(), name='job_detail'),
+    url(r'^$', JobListView.as_view(), name='job_list'),
+    url(r'^create/$', JobCreateView.as_view(), name='create_job'),
```

Comment:
Could we please reconsider the choice of the name `create_job` to be something more in line with the other URL names? `job_create` works, for instance.